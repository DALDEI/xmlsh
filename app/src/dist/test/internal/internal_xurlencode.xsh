# test of xurlencode

xurlencode http://www.xmlsh.org
xurlencode "a string & with < weird =? chars"  "second&second"
xurlencode -q "first" "arg1" "sec&end" "arg 2"
xurlencode -q <[ "a" , 1 , "b=2" , "spam&amp;bletch" ]>