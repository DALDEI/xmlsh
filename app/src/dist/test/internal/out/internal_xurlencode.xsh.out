http%3A%2F%2Fwww.xmlsh.org
a+string+%26+with+%3C+weird+%3D%3F+chars&second%26second
first=arg1&sec%26end=arg+2
a=1&b%3D2=spam%26bletch
